common: fuel_release_version param is added

rkhozinov opened the pull request:
Related to fuel-qa changes
https://github.com/openstack/fuel-qa/commit/96de67b04506d12047f29cad3d883753b49a425c

Commit:
common: properties parsing mechanism is fixed

Commit:
common: properties parsing mechanism is fixed

Commit:
9.x/nsxv: plugin version is fixed

Plugin version will be defined by NSXV_PLUGIN_VERSION
from SPECIFIC_PROPS

Commit:
9.x/nsxv: plugin version is fixed

Plugin version will be defined by NSXV_PLUGIN_VERSION
from SPECIFIC_PROPS

naumvd95 committed:
defining plugin-version fix

to avoid https://mirantis.jira.com/browse/FPL-2699

naumvd95 committed:
disconnect-on-failure fixe logic

naumvd95 committed:
common: disconnect-on-failure fixed logic

naumvd95 committed:
duplicated defining was removed

naumvd95 committed:
duplicated defining was removed

naumvd95 committed:
add separate logic for vcenter conv-snap

naumvd95 committed:
add separate logic for vcenter conv-snap

naumvd95 committed:
using right-way to string comparisons

Commit:
nsxv: stylefixes for bash

naumvd95 committed:
wrapping define-case

results=http://jenkins-tpi.bud.mirantis.net:8080/view/9.x.vcenter/job/9.x.vcenter.custom.qa/23/console

Commit:
common: fuel_release_version param is added

Related to fuel-qa changes
https://github.com/openstack/fuel-qa/commit/96de67b04506d12047f29cad3d883753b49a425c

Commit:
travis: shellcheck is added

Source: https://github.com/koalaman/shellcheck/wiki/TravisCI

naumvd95 commented on servers/partner-ci/common.yaml:
```diff
@@ -1,10 +1,14 @@
     parameters:
+      - string:
+          name: FUEL_RELEASE_VERSION
+          default: '9.0'
```
seems we need to override that with {fuel-release}?

rkhozinov replied:
I thought about it. But for that case we should create some sets of plugins jobs for each version of fuel I meant 9.0, 9.1, 9.2

rkhozinov replied:
Ideally, we could create
9.1 based on the `fuel-qa/9.1` branch
9.x base on the `fuel-qa/stable/mitaka`

naumvd95 replied:
and 1 more
there is no 9.0 version, as i know, so default value should be 9.1

rkhozinov replied:
```
 File "/home/jenkins/workspace/9.1.contrail.5.0.custom.dev/plugin_test/fuel-qa/fuelweb_test/tests/base_test_case.py", line 381, in check_fuel_version
    assert_equal(fuel_release, settings.FUEL_RELEASE_VERSION, err_msg)
AssertionError: Fuel version should be `9.2` but it is `9.0`
```

rkhozinov committed:
Revert "travis: shellcheck is added"

Commit:
vcenter,common: full path to the conv.py util

rkhozinov commented:
@naumvd95 stable/5.0.0 has been created by @ilush
and now we should create separate set of jobs to check the ref https://review.openstack.org/#/c/407129/

naumvd95 reviewed (changes requested):
need to check that
do u want to create separate jobs?
or remove current?

Commit:
9.1/contrail: some parameters are moved to SPECIFIC_PROPS

Commit:
nsxv: bash fixes after refactoring

Commit:
nsxv: bash fixes after refactoring

rkhozinov committed:
Revert "nsxv: bash fixes after refactoring"

Commit:
common: git log format is changed to oneline

Commit:
9.0/contrail: 9.0 is merged with 9.1

naumvd95 commented on servers/partner-ci/9.0/contrail.yaml:
```diff
@@ -21,30 +20,17 @@
+            CONTRAIL_DISTRIBUTION=;
+            CONTRAIL_PLUGIN_PACK_UB_PATH=;
+            MULTIPLE_NETWORKS=;
+            MULTIPLE_NETWORKS_TEMPLATE=;
```
seems that is useless?

rkhozinov replied:
they are used in tests, let's save it for custom

naumvd95 commented on servers/partner-ci/9.0/contrail.yaml:
```diff
@@ -21,30 +20,17 @@
     name: 'predefined_parameters'
     fuel-release: '9.1'
-    plugin-version: '5.0'
+    plugin-version: '5.0.0'
```
did u checked, that is no diffs with test-cases?

rkhozinov replied:
is it used in tests?

naumvd95 commented on servers/partner-ci/9.0/contrail.yaml:
```diff
@@ -271,7 +253,7 @@
 
       - '{regression-name}':
-          testrail_suite: "[9.x][PCE] Fuel plugin contrail v5.0.0 Regression"
+          testrail_suite: "[9.0][PCE] Fuel plugin contrail v5.0.0 Regression"
```
maybe we need to create separate regression suite or rename current?

rkhozinov replied:
we'll rename current

Commit:
9.x/contrail: new suite is added

naumvd95 commented on servers/partner-ci/common.yaml:
```diff
@@ -347,7 +347,11 @@
                 echo "snapshot_id_number=" $snapshots_id
+<<<<<<< HEAD
                 python "${CONV_SNAP_PATH}"
+=======
```
merge conflict!!!!!

naumvd95 committed:
global job-reconfiguring

1)separate jobs into 3 parts (9.0/9.1/9.x) by fuel_release_version var
2)use separate repo for each release
3)try to test custom fuel-qa repo

Commit:
9.x/vcenter: style related refactoring

Commit:
9.x/vcenter: vcenter vars refactoring

naumvd95 committed:
remove node_configs from common

naumvd95 committed:
code-style fixes

naumvd95 committed:
bug-fixes to pass jjb test

results https://mirantis.jira.com/browse/FPL-2707

naumvd95 committed:
replace DISABLE_SSL var to COMMON_PROPS

Commit:
all: fuel-release is renamed to fuel_release

naumvd95 committed:
all: pass fuel_release var into global common

Commit:
common: git log is cutted to the last 10 lines

Commit:
all: fuel_release is renamed for 9.x

naumvd95 committed:
bug-fix to pass tox-test

tox config changed to associate with often used jobs

rkhozinov commented on tox.ini:
```diff
@@ -33,6 +33,33 @@ commands =
    jenkins-jobs --conf {env:CONFIG} test {env:SERVERS}/7.0/vcenter.yaml:{env:COMMON}   -o {env:OUTPUT}
```
Remove old

rkhozinov replied:
Also need to add pip requirements

naumvd95 committed:
unfortunately there is no case to run different commands in overall dir

rkhozinov commented on tox.ini:
```diff
@@ -33,7 +25,7 @@ commands =
 
-[testenv:partner-ci]
+[testenv]
 #deps = -r{toxinidir}/conf/requirements.txt
```
restore that.
get reqs from the jenkins-jobs-infra

naumvd95 committed:
reorginize tox

Commit:
tox: requirements are fixed

Commit:
all: plugin_version variable is renamed to pluing_version
